src/extension.ts

```diff
@@ -35,7 +35,7 @@ export function activate(context: vscode.ExtensionContext) {
     disposable.push(
         vscode.commands.registerTextEditorCommand('local-history.removeRevision', () => {
             const editors = vscode.window.visibleTextEditors;
-            if (editors && editors.length > 0) {
+            if (editors && editors.length === 2) {
```

Comment:
Do you mind clarifying why we are checking if the length is exactly 2?

Reply:
length is exactly 2 when we are in diff mode since we have two editors visible side-by-side.

Reply:
I’m not sure we should do such a check, if we were to close the original file and keep only the diff would it still hold true? Or if we had multiple editors opened.

Reply:
I think this check would cause this issue https://github.com/vince-fugnitto/local-history-ext/issues/40. The issue no longer exists when I will change it back to `length > 0`.

Reply:
It still hold true when we kept only the diff or had multiple editors opened. I was still able to delete the revision.